src/popover/internal.tsx

```diff
@@ -26,6 +26,7 @@ import { useSingleTabStopNavigation } from '../internal/context/single-tab-stop-
 
 export interface InternalPopoverProps extends PopoverProps, InternalBaseComponentProps {
   __onOpen?: NonCancelableEventHandler<null>;
+  __triggerTypeFlex?: boolean;
```

Comment:
This change is needed so that a popover can be added to the new edit button of the grouped filtering token. Without that the popover trigger styles don't allow that custom button to grow and render correctly inside the token.

Reply:
I would suggest to extend `triggerType` internally instead (same way as `variant` for button: https://github.com/cloudscape-design/components/blob/main/src/button/internal.tsx#L40) and introduce a new type `groupedToken`, because `flex` does not say much about type of content, but rather describes style rule.

Reply:
Moreover, this approach allows you to get rid of the conditions in the code below